src/index.ts

```diff
@@ -125,13 +126,18 @@ async function main() {
       params,
     })
 
+    const returnType = getReturnType({
+      type: fnAst,
```

Comment:
I will probably change the name of this variable from `type` to something else later. `type` doesn't sound right, maybe `functionAST` out loud?

Doesn't block this PR though, will merge shortly.